code/modules/mob/living/simple_animal/hostile/terror_spiders/ghost.dm

```diff
@@ -40,6 +40,12 @@
 		to_chat(user, "<span class='notice'>Someone else already took this spider.</span>")
 		return
 	key = user.key
+	to_chat(src, "<b><font size=3 color='red'>You are a Terror Spider!</font></b>")
```

Comment:
Dupe code my beloathed. It is better to make a `give_introduction` proc which does all the text here. Then call that in both cases